A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/PFD/AirspeedIndicator.js

```diff
@@ -1021,6 +1045,15 @@ class Jet_PFD_AirspeedIndicator extends HTMLElement {
             takeOffSpeedNotSet = true;
         }
 
+        if (this.decelText) {
+            if (_isDecel) {
+                this.decelText.setAttribute("visibility", "visible");
+                this.decelText.textContent = "DECEL";
+            } else {
+                this.decelText.setAttribute("visibility", "hidden");
+            }
```

Comment:
Can we diff with the previous value for performance reasons ?

Reply:
yes we could, FYI the whole display doesn't do that atm